### PR TITLE
Improve info parsing and consistency check.

### PR DESCRIPTION
**Pull request by time-trader**
## Summary
Make several small improvements based on real-life testing by @time-trader.

### Enhancements
- No longer output timestamp check warnings in case of IMU sensor deviations to reduce log clutter
- Re-initialize previous timestamps when the wake info packet is received
- Exiting info packet parsing after the specific info packet is parsed

### Fixes
- Reinitialize `sensor_time_offset` upon handle update

### CODE REVIEW

File: data_gateway/packet_reader.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import datetime
+Startimport datetime
```
Comment (cortadocodes) [inline data_gateway/packet_reader.py:1]: There's an accidental "Start" added to the start of this line causing the tests to fail

File: data_gateway/packet_reader.py [hunk 2]
```diff
@@ -413,17 +413,28 @@ def _parse_info_packet(self, information_type, payload):
...
+            return
 
-        elif information_type == "Sleep State":
+        if information_type == "Sleep State":
             state_index = str(int.from_bytes(payload, self.config.endian, signed=False))
             logger.info("\n%s\n", self.config.sleep_state[state_index])
-            self.sleep = bool(int(state_index))
 
-        elif information_type == "Info Message":
+            if bool(int(state_index)):
```
Comment (cortadocodes) [inline data_gateway/packet_reader.py:427]: Can you explain why this has changed in the PR description?

**Review by cortadocodes — CHANGES_REQUESTED** [review]
Thanks! Please could you:
- [ ] Update the package version in `setup.py`
- [ ] Ensure the tests pass after addressing the other comments
- [ ] Remove unnecessary information from the auto-generated PR description (e.g. the merge commit details)
- [ ] Update the pull request description so that it describes the current state of the PR well

File: data_gateway/packet_reader.py [hunk 3]
```diff
@@ -470,18 +476,24 @@ def _check_for_packet_loss(self, sensor_name, timestamp, previous_timestamp):
...
             if abs(timestamp_deviation) > self.config.max_timestamp_slack:
-                logger.warning(
-                    "Possible packet loss. %s sensor packet is timestamped %d ms later than expected",
-                    sensor_name,
-                    timestamp_deviation * 1000,
-                )
+
+                if self.sleep:
+                    # Only Constat comes during sleep
+                    # TODO add different period for constat during sleep
```
[the remark this answers is not in the document]
Comment (time-trader) [reply data_gateway/packet_reader.py:485]: Not even sure if it is actually needed...

File: data_gateway/packet_reader.py (hunk 2 above)
Comment (time-trader) [reply data_gateway/packet_reader.py:429]: Done

**Pull request merged**